spec/requests/riiif_spec.rb

```diff
@@ -1,29 +1,31 @@
 RSpec.describe 'IIIF image API', type: :request do
   let(:user) { create(:user) }
-  let(:work) { create(:work_with_one_file, user: user) }
-  let(:file_set) { work.ordered_members.to_a.first }
-  let(:file) { file_set.original_file }
   let(:size) { '300,' }
 
-  before do
-    Hydra::Works::AddFileToFileSet.call(file_set,
-                                        File.open(fixture_path + '/world.png'),
-                                        :original_file)
+  let(:file_set) do
+    create_for_repository(:file_set,
+                          user: user,
+                          content: file)
+  end
+  let(:file) { fixture_file_upload('/world.png', 'image/png') }
```

Comment:
I'm getting `RuntimeError: /world.png file does not exist` here. Not sure if this has something to do with including `ActionDispatch::TestProcess`.

Reply:
I'm not seeing this error on my computer.